frontend/src/components/FilterDrawerContent/index.ts

```diff
@@ -0,0 +1 @@
+export { FilterDrawerContent } from './FilterDrawerContent'
```

Comment:
I've started just making components not wrapped in folders. Feel free to do the same if so inclined.   🙂